### PR TITLE
Attempt to fix Gzip failures.

### PR DESCRIPTION
**Pull request by Maikuolan**
(See comments at #340).

### CODE REVIEW

**Comment by Maikuolan** [comment]
StyleCI complaints. '-.-

One sec.

**Comment by Maikuolan** [comment]
Tests look good now. :-)

**Review by lablnet — APPROVED** [review]
👍

File: src/Archive/Adapter/Gzip.php [hunk 1]
```diff
@@ -84,7 +84,7 @@ public function compress($files, string $destination = '', bool $overwrite = fal
         }
 
         $filename = $destination;
-        if ($outfile = gzopen($filename, $mode)) {
+        if ($outfile = gzopen($filename, 'wb'.$mode)) {
```
Comment (peter279k) [inline src/Archive/Adapter/Gzip.php:87]: I think  it should verify the compression mode before calling the `gzopen` function.

According to the [gzcompress](https://www.php.net/manual/en/function.gzcompress.php) reference, the compression level range should be from `0` to `9`.

Comment (lablnet) [reply src/Archive/Adapter/Gzip.php:87]: > I think it should verify the compression mode before calling the `gzopen` function.

agreed :+1:

**Pull request merged**